language-support/qute/qute.tmLanguage.json

```diff
@@ -361,7 +393,7 @@
         }
       ]
     },
-    "keywords": {
+    "keywordsToRemove": {
```

Comment:
remove this keywordsToRemove rule.

Reply:
@AlexXuChen please remove 

 * the full rule keywordsToRemove and defines missing operator in the `"expression"` rule defined in https://quarkus.io/guides/qute-reference#built-in-resolvers (in 2 matches one for `or` in purple and one for `?:` in black)
 * the `expression_content` rule which is not used.